packages/calypso-e2e/src/lib/blocks/block-flows/instagram.ts

```diff
@@ -36,16 +35,12 @@ export class InstagramBlockFlow implements BlockFlow {
 			.getByPlaceholder( 'Enter URL to embed here…' )
 			.fill( this.configurationData.embedUrl );
 
-		if ( envVariables.TEST_ON_ATOMIC ) {
-			await editorCanvas.getByRole( 'button', { name: 'Embed' } ).click();
-		} else {
-			// For some reason, the hierarchy/selector changes for simple sites. Probably
-			// because it's running from within the Gutenframe?
-			await editorCanvas
-				.getByRole( 'document', { name: 'Block: Embed' } )
-				.getByRole( 'button', { name: 'Embed' } )
-				.click();
-		}
+		await editorCanvas
+			.getByRole( 'document', { name: 'Block: Embed' } )
+			.getByRole( 'button', {
+				name: 'Embed',
+			} )
+			.click();
```

Comment:
I've verified that this selector works for both AT and Simple, on Gutenberg Production and Edge.